MIAUI/ViewModels/SubtasksViewModel.cs

```diff
@@ -1,25 +1,38 @@
 ﻿namespace MIAUI.ViewModels;
 
 [INotifyPropertyChanged]
+[QueryProperty("TaskName", "TaskName")]
 public partial class SubtasksViewModel
 {
+
+    public SubtasksViewModel()
+    {
+        Details = new ObservableCollection<string>();
```

Comment:
We will probably want to update this to be an ObservableCollection<SubTasks> but this can happen when you all start setting up the database!

Reply:
I'm leaving this alone for now until the database is set up or in the process of being set up :)